crates/sqlexec/src/session.rs

```diff
@@ -493,6 +495,7 @@ impl Session {
     pub async fn execute_inner(
         &mut self,
         plan: LogicalPlan,
+        query_text: &str,
```

Comment:
this logicalplan is already a wrapper around the datafusion type, so it could become a tuple of the query-text (or empty string) and the LogicalPlan and that way the function signatures don't need to change.

Reply:
I tried doing that initially. I can't remember what made me change my mind and do this instead. Anyways, will update!

Reply:
While trying to do the same, I realised these two combine and disassociate quickly. They are required together here only. Combining them makes the code elsewhere hard to work with. Yes, this function signature feels tacky, but we can fix that.